velox/core/PlanNode.h

```diff
@@ -564,6 +576,14 @@ class AggregationNode : public PlanNode {
     return ignoreNullKeys_;
   }
 
+  const std::vector<vector_size_t>& globalGroupingSets() const {
+    return groupingSets_.globalGroupingSets;
+  }
+
+  std::optional<std::string> groupIdChannel() const {
```

Comment:
channel is the index of a column in the RowVector, hence, cannot be of type std::string. I suggest to change this to 

```
FieldAccessTypedExprPtr groupId() const;
```